src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -921,6 +1031,37 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
           write(6,*)'fv3lam_io_dynmetvars2d_nouv is ',(trim(fv3lam_io_dynmetvars2d_nouv(i)),i=1,ntracerio3d)
       endif      
 
+    if (laeroana_fv3cmaq) then
```

Comment:
Please align line 1034 with line 1032 and ensure two-space indentation for all if statements and do loops.